_partials/cli_tools.md

```diff
@@ -12,7 +12,11 @@ Let's install them, along with other useful tools:
 
 ```bash
 sudo apt update
-sudo apt install -y vim tmux tree git ca-certificates curl jq unzip zsh apt-transport-https gnupg software-properties-common direnv sqlite3 make
+sudo apt install -y vim tmux tree git ca-certificates curl jq unzip zsh \
```

Comment:
It would be amazing to break down the lines per "groups" of tools, like 
```suggestion
sudo apt install -y vim tmux tree git ca-certificates curl jq unzip zsh # basic bash navigation tools
sudo apt-install ... # python installation dependencies
sudo apt-install ... # docker & kube etc...
...
```